src/Components/Server/src/Circuits/DefaultCircuitFactory.cs

```diff
@@ -5,31 +5,33 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Security.Claims;
-using System.Text.Encodings.Web;
 using Microsoft.AspNetCore.Components.Routing;
 using Microsoft.AspNetCore.Components.Web.Rendering;
-using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Microsoft.JSInterop;
 
 namespace Microsoft.AspNetCore.Components.Server.Circuits
 {
-    internal class DefaultCircuitFactory : CircuitFactory
+    internal class DefaultCircuitFactory
```

Comment:
rename this to circuit factory if you're removing the base class 😆

Reply:
done